artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/proton/AMQPConnectionContext.java

```diff
@@ -79,7 +87,7 @@
 
    protected final ProtonHandler handler;
 
-   protected AMQPConnectionCallback connectionCallback;
+   public AMQPConnectionCallback connectionCallback;
```

Comment:
Does this need to be public? Seems odd when nothing else is.

Reply:
thanks again. nice catch.